ra-front/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "local": "ng serve --port 4444 --host 0.0.0.0 --disable-host-check"
```

Comment:
This might be helpful, running locally will only require `npm run local` instead of `ng serve --port 4444 --host 0.0.0.0 --disable-host-check`